spec/classes/katello_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
```

Comment:
Nitpick: typically the test name follows the full manifest name as a convention, i.e. `certs_katello_spec.rb`